src/components/mixins/currencySymbol.js

```diff
@@ -0,0 +1,16 @@
+const symbols = {
+  USD: '$',
+  RUB: '₽',
+  EUR: '€'
+}
+
+const currencySymbol = {
+  computed: {
+    getSymbol () {
```

Comment:
I think just "symbol" or "currencySymbol" or like is better because the name "getSymbol" sounds like a function but actually it is used as if it was just a property. What do you think? 🤔 
```
this.getSymbol() // sounds like this, but
this.getSymbol // it is how it is used actually
```

Reply:
Yeah, full agree with, I will change it